js/main.js

```diff
@@ -25,6 +25,7 @@ CRM.$(function($) {
     function skipPaymentMethod() {
       var flag = false;
       $('.price-set-option-content input').each( function(){
+        if (!$(this)[0].hasAttribute("data-amount")) return;
```

Comment:
I feel like it should be 
```
f (!$(this).hasAttribute("data-amount")) return;
```

without the zero?

Reply:
with you changes, getting JS error
```
main.js?qghgm2:28 Uncaught TypeError: $(...).hasAttribute is not a function
```
added 3 console log for debugging
```
        console.log($(this).attr('id'))
        console.log($(this));
        console.log($(this)[0]);
```
<img width="845" alt="Screenshot 2020-09-11 at 3 59 24 PM" src="https://user-images.githubusercontent.com/377735/92912966-f0eada80-f447-11ea-8a8e-ee9e96c3ec46.png">

`$(this)` have 2 element and `hasAttribute` is not jQuery function.